insecure/attacker.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package corruptible;
+
+import "google/protobuf/empty.proto";
+
+option go_package = "insecure";
+
+// to compile this file, run:
+// protoc insecure/attacker.proto --go_out=plugins=grpc:.
+
+// CorruptibleConduitFactory allows an attacker to register itself to the factory, and send messages on behalf of its conduits.
+service CorruptibleConduitFactory {
+  rpc RegisterAttacker(AttackerRegisterMessage) returns (google.protobuf.Empty) {}
+  rpc ProcessAttackerMessage(Message) returns (google.protobuf.Empty) {}
+}
+
+service Attacker {
+  rpc Observe(Message) returns (google.protobuf.Empty) {}
```

Comment:
shouldn't this return a [stream](https://grpc.io/docs/languages/go/basics/#server-side-streaming-rpc) instead? That way the Attacker would keep getting all the messages coming on the conduit?

Reply:
https://github.com/onflow/flow-go/pull/2033/commits/b4f979db4706d4e020341841de24712364a6596a